graylog2-web-interface/src/views/components/sidebar/SectionInfo.jsx

```diff
@@ -0,0 +1,9 @@
+// @flow strict
+import styled, { type StyledComponent } from 'styled-components';
+import { type ThemeInterface } from 'theme';
+
+const Title: StyledComponent<{}, ThemeInterface, HTMLParagraphElement> = styled.p(({ theme }) => `
+  color: ${theme.colors.gray[60]};
```

Comment:
IMO this is too light for anyone who is even slightly reading impaired. We should not use more than 50 for text, in this case 40 is even better.

Reply:
That is true, I am now using the default text color.